TKGame/TKGame.cs

```diff
@@ -288,5 +294,16 @@ protected override void Draw(GameTime gameTime)
             // Once everything has finished drawing, figure out the framerate
             GameDebug.UpdateFPS(gameTime);
         }
+
+        public void ExitGame()
+        {
+            LevelEditor.SaveStageDataToJSON(currentStage, "auto_saved_stage_data");
+            Exit();
+        }
+
+        public static void SwitchToGameplayMenu() 
+        {
+            Instance.desktop.Root = Instance.VSP;
```

Comment:
so desktop.Root is basically the main screen input? I'm not sure VSP is the best name for the GameplayMenu assuming this changes from menu to gameplay. Additionally, if this is switching that, do we not have a way to get back or is that just being done upon pausing now? In which case The level editor will need to be modified since it pauses the game.

Reply:
VSP was only used in the debug stuff before the weapon system UI stuff was added, so i wasn't expecting it to be used outside of that class lol. will be fixed with refactoing for sure. we can store like a grid or vertical stack panel in a class and swap between them as necessary.